pyproject.toml

```diff
@@ -9,6 +45,9 @@ show_missing = true
 minversion = "6.0"
 log_cli_level = "INFO"
 asyncio_mode = "auto"
+markers = [
+  'setup: marks tests as setup (deselect with -m "not setup")'
+]
```

Comment:
- What is this marker for?
- What's the effect of having makers listed here?

Reply:
The existing set of tests had the `setup` marker to imply that these are setup steps:

``` python
@pytest.mark.setup
async def test_build_and_deploy(ops_test: OpsTest):
```
Without adding:
```
markers = [
  'setup: marks tests as setup (deselect with -m "not setup")'
]
```
You get these warnings:
```
PytestUnknownMarkWarning: Unknown pytest.mark.setup - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.setup
```
With them we can run:
`make integration -- -m "setup"` or `make integration -- -m "not setup"`